contents/newsletter/small-teams.md

```diff
@@ -0,0 +1,116 @@
+
+### 5. They’re flexible
+
+We’d rather hire new people than keep moving people around to fill gaps. That said, we’re happy for people to move between teams when needed, ideally no more often than every three to nine months. There are two scenarios that could trigger a move:
+
+* A small team may realize they no longer need someone, or that they could use the help of someone internally on another small team.
+* Someone might want to move to another small team to develop their skills or experience.
+
+Rather than the team lead making the call, it’s always the manager who decides if one of their direct reports should move between teams. But, once in a small team, you’re only in one team. Someone being in multiple small teams at once defeats the purpose of ownership.
+
+If someone feels the need to be in more than one small team, that likely means we need to hire. That can mean we’re ‘overstaffed’ at times, but ultimately good people will do good work to push our product forward. And it’s better to have slightly overstaffed small teams than to be perpetually understaffed, and for your product to suffer as a result.
+
+Small teams are also able to work with other small teams on multidisciplinary projects, and members of one small team can and should attend meetings on other small teams to help with collaboration if needed.
+
+### The tradeoffs of small teams
+
+Small teams aren’t perfect. And sometimes, to make them work, we’ve had to create workarounds and accept some tradeoffs.
+
+- **They overlap.** Some overlap is inevitable, especially when teams work on features that are used across multiple products. We mitigate this by being [aggressively transparent](/founders/how-to-run-a-transparent-company). We also have a list of everything we do [and who owns it](/handbook/engineering/feature-ownership), so it's easy to see who someone should talk to about a shared problem.
+
+- **Fuzzy ownership.** This isn’t unique to small teams, but we still mitigate against it. If a product or project doesn't have a clear owner, our [fuzzy ownership process](/handbook/company/fuzzy-ownership) encourages people to create a PR and find a solution. If a problem is big enough and it doesn't have owner, we sometimes form a temporary team just to solve that problem. We also [actively encourage](https://posthog.com/handbook/values) people to step on toes in a low-ego way. 
+
+- **Speed over seamlessness.** Small teams allow us to ship fast. But the tradeoff here is that we tolerate some level of not-so-great integration in exchange for speed. This isn't a problem unique to PostHog. AWS is a great example of a very, very big company that favours speed over integration.
+
+- **Add more small teams!** We want to avoid having big, slow teams. In doing so, we’ve kept adding more small teams. This is a hypothetical concern, but creating too many small teams could bottleneck each other. 
```

Comment:
```suggestion
- **Bottlenecks** We want to avoid having big, slow teams. In doing so, we’ve kept adding more small teams. This is a hypothetical concern, but creating too many small teams could bottleneck each other. 
```